tangelo/algorithms/classical/mp2_solver.py

```diff
@@ -83,7 +83,17 @@ def simulate(self):
         if self.uhf:
             self.mp2_fragment = self.mp.UMP2(self.mean_field, frozen=self.frozen)
         else:
-            self.mp2_fragment = self.mp.RMP2(self.mean_field, frozen=self.frozen)
+            import pyscf
+            if pyscf.__version__ == '2.5.0' and self.mean_field.istype('ROHF'):
```

Comment:
I was expecting `>= 2.5.0`, I think we can assume that the change is here to last over newer versions as well

Reply:
The thing is, pyscf made a mistake in the code in version 2.5 which has been fixed now but the fix will be available only from the later version so this issue is very specific to the 2.5 version after which we wont need this fix

Reply:
That's true, I forgot you mentioned that, thank you.

Reply:
Yehh